src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/change_point/fields_suggestions_after.test.ts

```diff
@@ -0,0 +1,262 @@
+        name: 'count',
+        type: 'column',
+      },
+      {
+        name: 'on',
+        args: [
+          {
+            args: [
+              {
+                name: 'field1',
+                location: {
+                  min: 152,
+                  max: 160,
+                },
+                text: 'field1',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 152,
+              max: 168,
+            },
+            text: 'field1',
+            incomplete: false,
+            parts: ['field1'],
+            quoted: false,
+            name: 'field1',
+            type: 'column',
+          },
+        ],
+        location: {
+          min: 149,
+          max: 168,
+        },
+        text: '',
+        incomplete: false,
+        type: 'option',
+      },
+      {
+        name: 'as',
+        args: [
+          {
+            args: [
+              {
+                name: 'changePointType',
+                location: {
+                  min: 173,
+                  max: 187,
+                },
+                text: 'changePointType',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 173,
+              max: 187,
+            },
+            text: 'changePointType',
+            incomplete: false,
+            parts: ['changePointType'],
+            quoted: false,
+            name: 'changePointType',
+            type: 'column',
+          },
+          {
+            args: [
+              {
+                name: 'pValue',
+                location: {
+                  min: 190,
+                  max: 195,
+                },
+                text: 'pValue',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 190,
+              max: 195,
+            },
+            text: 'pValue',
+            incomplete: false,
+            parts: ['pValue'],
+            quoted: false,
+            name: 'pValue',
+            type: 'column',
+          },
+        ],
+        location: {
+          min: 170,
+          max: 195,
+        },
+        text: '',
+        incomplete: false,
+        type: 'option',
+      },
+    ],
+    location: {
+      min: 130,
+      max: 195,
+    },
+    text: 'CHANGE_POINTcountONextension.keywordASchangePointType,pValue',
+    incomplete: false,
+    type: 'command',
+  } as unknown as ESQLAstCommand;
+
+  it('should return the correct fields after the command without AS option', () => {
+    const previousCommandFields = [
+      { name: 'field1', type: 'keyword' },
+      { name: 'count', type: 'double' },
+    ] as ESQLRealField[];
+
+    const userDefinedColumns = [] as ESQLRealField[];
+
+    const result = fieldsSuggestionsAfter(
+      changePointCommandWithoutAs,
```

Comment:
I wonder if this object AST snapshot can be replaced with the [*synth* API](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-esql-ast/src/synth/README.md)?

```suggestion
      synth.cmd `CHANGE_POINT count ON field1`,
```

Reply:
Checking!

Reply:
Great comment Vadim. I forgot our synth api 😍 Done here https://github.com/elastic/kibana/pull/218242/commits/1b2e211bc8f93becf7035daae4d04e3970400a96